src/lib/CalendarAPI.ts

```diff
@@ -1,20 +1,35 @@
-import axios from 'axios';
+import axios, { AxiosRequestConfig } from 'axios';
 
 import Calendar from "@/models/Calendar";
 import CalendarEvent from "@/models/CalendarEvent";
 import User from "@/models/User";
 
+function jsonCleaner(key: any, value: any): any {
+  // Remove the pseudo-elements that are stored in the model
+  // classes here, but should not be serialized back to the API
+  if (key == "calendar" || key == "events" || key == "calendars") {
+    return undefined;
+  }
+  return value;
+}
+
+function toJson(obj: any): string {
+  return JSON.stringify(obj, jsonCleaner);
+}
+
+const config: AxiosRequestConfig = { headers: {'Content-Type': 'application/json'} };
+
 // TODO
-const API_ROOT: string = process.env.VUE_APP_API_ROOT || "http://localhost:8080/api/";
+const API_ROOT: string = process.env.VUE_APP_API_ROOT || "http://api:8080/";
```

Comment:
May need to change at a later time because the api's port is 6868. should be good to keep for now.